library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -1776,6 +1788,7 @@ public StateBuilder edit() {
         private int firstDayOfWeek = Calendar.getInstance().getFirstDayOfWeek();
         public CalendarDay minDate = null;
         public CalendarDay maxDate = null;
+        public boolean saveCurrentPosition = false;
```

Comment:
Good that it is included in the StateBuilder as well, to match several other parameters.